### PR TITLE
AVI-2: Update readme with markdown rich code formatting

### PR DESCRIPTION
**Pull request by Doyin02**
made some few changes on the readme file to make the file look better

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,18 +19,21 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone 
+
+`https://github.com/harry-olabs/avi-hub.git`
+
```
Comment (harry-olabs) [inline README.md]: This should be together on a new line. Something like this:

```
git clone https://github.com/harry-olabs/avi-hub.git
```

This is a code block format ☝️. If you hover over it, GitHub allows you to click copy button to copy the command.

File: README.md [hunk 2]
```diff
@@ -19,18 +19,21 @@ To run the **HR Management** Web App, ensure your system meets the following req
...
 * Navigate to the project directory: 
 cd hr-management-web-app
 
 ## Install dependencies using npm 
 
-* *npm* install or *yarn* install
+* `npm install` or `yarn install`
```
Comment (harry-olabs) [inline README.md]: For simplicity, let's stick with npm. Remove `yarn` commands and make this a block code format

File: README.md [hunk 3]
```diff
@@ -19,18 +19,21 @@ To run the **HR Management** Web App, ensure your system meets the following req
...
 
 ## Start the development server:
 
-*npm* start
-*yarn* start
+`npm start` or 
+`yarn start`
```
Comment (harry-olabs) [inline README.md]: Same here

**Review by harry-olabs — CHANGES_REQUESTED** [review]
Good job @Doyin02, just a couple comments and we should be good to go 🚀 

As a related note, please update PR title in the format {ticket_name}: {ticket_title}
For example: `AVI-2: Add support for magicians`

File: README.md [hunk 4]
```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
...
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone ```https://github.com/harry-olabs/avi-hub.git```
+
 * Navigate to the project directory: 
 cd hr-management-web-app
 
 ## Install dependencies using npm 
 
-* *npm* install or *yarn* install
+* ```npm install```
```
Comment (harry-olabs) [inline README.md]: Please read the GitHub documentation again to see to format code blocks:

Take a look at [Quoting Code](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)

File: README.md [hunk 5]
```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
...
 
 ## Start the development server:
 
-*npm* start
-*yarn* start
+```npm start```
```
Comment (harry-olabs) [inline README.md]: Update this after reading the documentation

File: README.md [hunk 6]
```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone ```https://github.com/harry-olabs/avi-hub.git```
```
Comment (harry-olabs) [inline README.md]: Like I mentioned before, this line is an entire command code. Wrap it using the format in the documentation.

Spend some time to read the documentation properly use code blocks format (triple backticks on separate lines).

**Review by harry-olabs — CHANGES_REQUESTED** [review]
Please spend some time to read the documentation

File: README.md [hunk 7]
```diff
@@ -19,18 +19,18 @@ To run the **HR Management** Web App, ensure your system meets the following req
...
+
 * Navigate to the project directory: 
 cd hr-management-web-app
```
Comment (harry-olabs) [inline README.md]: This should ideally be `cd avi-hub` and please wrap with code block

**Comment by Doyin02** [comment]
I have updated the README file

**Review by harry-olabs — APPROVED** [review]
Great job, @Doyin02. Looks good to me 🚀

**Pull request merged**